lib/ig.js

```diff
@@ -428,8 +445,9 @@ ${match[1]}
   };
   fs.copySync(patchPath, sdPath, { filter: filterFunc });
 
-  const hideSupporting = config.implementationGuide.primarySelectionStrategy
-  && config.implementationGuide.primarySelectionStrategy.hideSupporting;
+  const hideSupporting = (config.implementationGuide.primarySelectionStrategy
+  && config.implementationGuide.primarySelectionStrategy.hideSupporting)
+  || config.implementationGuide.hideSupporting;
```

Comment:
I'm assuming this change is because we are hoping to remove the `implementationGuide.primarySelectionStrategy` section of the config file entirely? If that's the case, could you update the `shr-cli` PR for this work to change the README to reflect the expected changes to the config file?

Reply:
You are correct. I updated the README to reflect this new flag, and also added some additional notes on the new usage of content profile. The changes are at https://github.com/standardhealth/shr-cli/pull/225/commits/08e4b2851cbebff14c09c824e01234d1240b64fe.